libbeat/tests/compose/compose.go

```diff
@@ -19,74 +19,134 @@ package compose
 
 import (
 	"errors"
+	"fmt"
 	"os"
 	"path/filepath"
 	"strconv"
+	"strings"
 	"testing"
 )
 
+// HostInfo exposes information about started scenario
+type HostInfo interface {
+	Host() string
+	HostForPort(int) string
+}
+
 // EnsureUp starts all the requested services (must be defined in docker-compose.yml)
 // with a default timeout of 300 seconds
-func EnsureUp(t *testing.T, services ...string) {
-	EnsureUpWithTimeout(t, 60, services...)
+func EnsureUp(t testing.TB, service string) HostInfo {
+	t.Helper()
+	return EnsureUpWithTimeout(t, 60, service)
 }
 
 // EnsureUpWithTimeout starts all the requested services (must be defined in docker-compose.yml)
 // Wait for `timeout` seconds for health
-func EnsureUpWithTimeout(t *testing.T, timeout int, services ...string) {
```

Comment:
There are two main reasons:
* It simplifies the returned information. With an only service, only an object with the host information for a service is returned.
* It doesn't seem to be needed, we only had an scenario with two services (Kibana, that also needs Elasticsearch), and it can also be done with docker compose dependencies. If access to both hosts were needed, we could have two `EnsureUp`.

If wouldn't be hard to implement it though, by returning a struct with all the services info and adding a method like `HostForServicePort(service, port) HostInfo`. But I didn't consider it neccessary by now.

Reply:
That sounds good to me, thanks for the explanation!